ci/docker/docker-compose.yml

```diff
@@ -66,10 +66,21 @@ services:
       - ./be-dataapp_data_receiver:/data
       - ./ecc_cert:/cert
 
+  uc-dataapp-provider:
+    image: rdlabengpa/ids_uc_data_app_platoon:v1.3
+    environment:
+      - TZ=Europe/Rome
+    expose:
+      - "8085"
```

Comment:
I would suggest that this port is not exposed put perhaps just internaly mapped to another one, to avoid overlaping with consumer UC. See the official TC repo docker-compose.

Reply:
On second thought, you actually don't need any of that, since we won't access it externaly ever.